docker-compose.platform.yml

```diff
@@ -35,9 +35,10 @@ services:
     depends_on:
       - drive_abci
     ports:
-      - 26656:26656 # P2P
+      - ${PLATFORM_DRIVE_TENDERMINT_P2P_PORT:?err}:26656 # P2P
```

Comment:
We need RPC port too (like in `master`):
- 127.0.0.1:${PLATFORM_DRIVE_TENDERMINT_RPC_PORT}:26657

Reply:
Added